### PR TITLE
ユーザー登録＋注文時のエラー文とプレースホルダーを修正

### PR DESCRIPTION
**Pull request by uedamiku**

### CODE REVIEW

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 1]
```diff
@@ -45,6 +45,10 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
 
 		User exitUser = registerUserService.searchByEmail(form.getEmail());
 
+		if (("").equals(form.getEmail())) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:48]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:48]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 2]
```diff
@@ -45,6 +45,10 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
...
+			result.rejectValue("email", null, "メールアドレスは必須です");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:49]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:49]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/RegisterUserController.java [hunk 3]
```diff
@@ -45,6 +45,10 @@ public String registerUser(@Validated RegisterUserForm form, BindingResult resul
...
+		}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/RegisterUserController.java:50]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 4]
```diff
@@ -34,13 +35,17 @@ public class SendOrderController {
 	 * @return 注文完了画面
 	 */
 	@PostMapping("/send-order")
-	public String sendOrder(@Validated SendOrderForm form,BindingResult rs, Model model) {
-		if(rs.hasErrors()) {
+	public String sendOrder(@Validated SendOrderForm form, BindingResult rs, Model model) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:38]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 5]
```diff
@@ -34,13 +35,17 @@ public class SendOrderController {
...
+
+		if (("").equals(form.getDestinationEmail())) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 6]
```diff
@@ -34,13 +35,17 @@ public class SendOrderController {
...
+			rs.rejectValue("destinationEmail", null, "メールアドレスは必須です");
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' child has incorrect indentation level 24, expected level should be 6.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 7]
```diff
@@ -34,13 +35,17 @@ public class SendOrderController {
...
+		}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:42]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:42]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if rcurly' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 8]
```diff
@@ -34,13 +35,17 @@ public class SendOrderController {
...
+		if (rs.hasErrors()) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:43]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'if' has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/SendOrderController.java [hunk 9]
```diff
@@ -34,13 +35,17 @@ public class SendOrderController {
...
 			return confirmOrderController.toConfirmOrder(form.getId(), model, form);
 		}
 		Order order = sendOrderService.sendOrder(form);
-		model.addAttribute("order",order);
-		
+		model.addAttribute("order", order);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/SendOrderController.java:47]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/form/RegisterUserForm.java [hunk 10]
```diff
@@ -41,8 +38,7 @@ public class RegisterUserForm {
 	@NotBlank(message = "住所は必須です")
 	private String address;
 	/** 電話番号 */
-	@NotBlank(message = "電話番号は必須です")
-	@Pattern(regexp = "^\\d{2,4}-\\d{2,4}-\\d{4}$", message = "電話番号はXXXX-XXXX-XXXXの形式で入力してください")
+	@Pattern(regexp = "^\\d{2,4}-\\d{2,4}-\\d{4}$", message = "電話番号はXXX-XXXX-XXXXの形式で入力してください")
```
Comment (tkeshun) [inline src/main/java/com/example/form/RegisterUserForm.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/form/RegisterUserForm.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 8, expected level should be 2.

**Review by yousuke-murayama — APPROVED** [review]
問題ないと思います。

**Pull request merged**